diff --git a/client/src/App.css b/client/src/App.css
index e658fa2..96cbe16 100755
--- a/client/src/App.css
+++ b/client/src/App.css
@@ -39,6 +39,7 @@
 
 .markerSymbol {
   position: absolute;
+  cursor: pointer;
 }
 
 .markerInfo {
@@ -144,6 +145,7 @@ p {
   max-width: 100%;
   padding-left: 1vh;
   padding-right: 1vh;
+  cursor: pointer;
 }
 
 .button:hover {
@@ -220,6 +222,7 @@ p {
   margin: 3px;
   padding: 1vh;
   z-index: 101;
+  cursor: pointer;
 }
 
 .switch {
diff --git a/client/src/marker.js b/client/src/marker.js
index 60ba3f7..40e20c0 100644
--- a/client/src/marker.js
+++ b/client/src/marker.js
@@ -106,6 +106,7 @@ export default class Mark extends React.Component {
                 viewBox="0 0 16 16"
                 fill="currentColor"
                 xmlns="http://www.w3.org/2000/svg"
+                cursor="pointer"
               >
                 <path
                   fillRule="evenodd"
diff --git a/client/src/submit.js b/client/src/submit.js
index e058afc..d0f534f 100644
--- a/client/src/submit.js
+++ b/client/src/submit.js
@@ -93,6 +93,7 @@ export default class Submit extends React.Component {
               viewBox="0 0 16 16"
               fill="currentColor"
               xmlns="http://www.w3.org/2000/svg"
+              cursor="pointer"
             >
               <path
                 fillRule="evenodd"
